src/Blocks/components/form/assets/form.js

```diff
@@ -1658,6 +1661,46 @@ export class Form {
 		this.utils.setOnFocus(event.target);
 	};
 
+	/**
+	 * On keypress event for regular fields.
+	 *
+	 * @param {object} event Event callback.
+	 *
+	 * @returns {void}
+	 */
+	onKeyDownEvent = (event) => {
+		const formId = this.state.getFormIdByElement(event.target);
+		const field = this.state.getFormFieldElementByChild(event.target);
+		const name = field.getAttribute(this.state.getStateAttribute('fieldName'));
+
+		if (this.state.getStateElementTypeCustom(name, formId) === 'number') {
+			const allowedKeys = [
```

Comment:
How would this hold against the [Chinese numerals](https://en.wikipedia.org/wiki/Chinese_numerals) ?

Reply:
I have no idea really and no way to test this :(

Reply:
Seems like Chrome (form > input field) does not support these.
From what I have found it seems that Arabic numbers are also a common method for numeric inputs in Chinese language

I'd say this is more than acceptable 😄